src/utils/booleans.ts

```diff
@@ -0,0 +1,19 @@
+import * as t from "io-ts";
+
+export type BooleanFromString = t.Type<boolean, string, unknown>;
+
+export const BooleanFromString: BooleanFromString = new t.Type<
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏What if it's renamed `FeatureFlag`, and implements itself the _default to false_ logic?

Reply:
Hi balanza, WB. Can you explain in more details what're you meaning?

Reply:
Thanks @balanza, I think I don't understand what you mean too

Reply:
Mmm maybe I get it.
Do you mean to drop the use of the `BooleanFromString` codec to have a `boolean|undefined` flag where the `undefined` means `false`?
If so, I'd say it's equivalent, maybe more explicit how it's implemented.

Reply:
Good point